### PR TITLE
[DOCS] Connecting to GCS using Spark

### PR DESCRIPTION
**Pull request by alexsherstinsky**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- JIRA: GREAT-187/GREAT-227
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [ ] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [ ] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: docs/guides/connecting_to_your_data/cloud/gcs/spark.md [hunk 1]
```diff
@@ -2,4 +2,153 @@
...
+
+<Tabs
+  defaultValue='runtime_batch_request'
+  values={[
+  {label: 'Specify a GCS path to single CSV', value:'runtime_batch_request'},
+  {label: 'Specify a data_asset_name', value:'batch_request'},
+  ]}>
+  <TabItem value="runtime_batch_request">
+
+Add the GCS path to your CSV in the `path` key under `runtime_parameters` in your `BatchRequest`.
```
Comment (anthonyburdi) [inline docs/guides/connecting_to_your_data/cloud/gcs/spark.md]: Just a tweak to be consistent with the class used in the example
```suggestion
Add the GCS path to your CSV in the `path` key under `runtime_parameters` in your `RuntimeBatchRequest`.
```

File: docs/guides/connecting_to_your_data/cloud/gcs/spark.md [hunk 2]
```diff
@@ -2,4 +2,153 @@
...
+<Congratulations />
+
+## Additional Notes
+
+<SparkAdditionalNotes />
+
+To view the full scripts used in this page, see them on GitHub:
+
+- [spark_s3_yaml_example.py](https://github.com/great-expectations/great_expectations/blob/develop/tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_yaml_example.py)
+- [spark_s3_python_example.py](https://github.com/great-expectations/great_expectations/blob/develop/tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_python_example.py)
```
Comment (anthonyburdi) [inline docs/guides/connecting_to_your_data/cloud/gcs/spark.md]: These should link to the gcs example docs

**Review by anthonyburdi — COMMENTED** [review]
Looks great! Just two tiny nitpicks.

**Comment by cdkini** [comment]
Haven't gotten a chance to take a detailed look but can we add any relevant links to the GCS docs? Please see the Pandas doc I wrote for reference!

Will take a look later today

File: docs/guides/connecting_to_your_data/cloud/gcs/spark.md (hunk 1 above)
Comment (alexsherstinsky) [reply docs/guides/connecting_to_your_data/cloud/gcs/spark.md]: Fixed (in next commit).  Thanks!

File: docs/guides/connecting_to_your_data/cloud/gcs/spark.md (hunk 2 above)
Comment (alexsherstinsky) [reply docs/guides/connecting_to_your_data/cloud/gcs/spark.md]: Great catch -- fixed!

**Review by anthonyburdi — APPROVED** [review]
LGTM!

**Pull request merged**